app/controllers/host/vms_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Host::VmsController < ApplicationController
```

Comment:
This is actually very interesting. There already is a `new` action `ComputeResourcesVmsController` - but there is no route for that. Weird...

We could just use that? What do you think, @dLobatog & @ohadlevy?